### PR TITLE
First kind-of working version

### PR DESCRIPTION
**Pull request by jkrems**
Mostly to ensure it's possible to wire the different bits together.

### CODE REVIEW

**Comment by EndangeredMassa** [comment]
Will this switch back to mocha as well?

**Comment by jkrems** [comment]
Yes, I think so. Keeping it consistent. Also while I'd like to implement different test frameworks eventually, I think for now just implementing `-wd` is enough work.

File: lib/browser/navigation.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        throw err;
+      }
+    });
+  });
+}
+
+var BASIC_PROPERTIES = [
+  'protocol', 'host', 'pathname', 'hash',
+];
+function urlAsserter(url, query) {
```
Comment (jkrems) [inline lib/browser/navigation.js]: This should be functionally equivalent to https://github.com/testiumjs/testium-driver-sync/blob/master/lib/browser/makeUrlRegExp.js.

File: test/integration/element.test.js [hunk 2]
```diff
@@ -0,0 +1,297 @@
+import {getBrowser} from '../mini-testium-mocha';
+import assert from 'assertive';
+
+function stripColors(message) {
+  return message.replace(/\u001b\[[^m]*m/g, '');
+}
+
+xdescribe('element', () => {
```
Comment (khoomeister) [inline test/integration/element.test.js]: Is this meant to be disabled?

File: test/integration/element.test.js [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+
+  it('can get an element\'s text', () => {
+    const element = browser.getElement('h1');
+    const text = element.get('text');
+    assert.equal('Element text was not found', 'Test Page!', text);
+  });
+
+  it('can get special properties from an element', () => {
+    // the "checked" property (when it doesn't exist)
+    // returns a non-standard response from selenium;
```
Comment (khoomeister) [inline test/integration/element.test.js]: I'm not sure what is meant by "non-standard response" - can you elaborate?

Comment (jkrems) [inline test/integration/element.test.js]: Erm, good question. This is copied from `testium-driver-sync` where it was copied from `testium`. I might have more to say about this when I get to porting this file, not sure yet how `wd` and `webdriver-http-sync` handle it. :)

File: test/integration/element.test.js (hunk 2 above)
Comment (jkrems) [inline test/integration/element.test.js]: Yep, all but `navigation` and `cookie` are pending (because only the navigation and cookie APIs are ported yet).

**Comment by khoomeister** [comment]
I guess all the `xdescribe`s are there for a good reason right now.

:shipit:

**Pull request merged**